src/components/data/Table.tsx

```diff
@@ -30,6 +27,7 @@ export default function Table({
   interactive = false,
   stickyHeader = false,
   borderless = false,
+  variation = 'striped',
```

Comment:
The `striped` term is after how Bootstrap names it https://getbootstrap.com/docs/5.3/content/tables/#striped-rows, but I'm open to other suggestions